<ui version="4.0" >
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qmmp Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="contentsWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize" >
        <size>
         <width>38</width>
         <height>38</height>
        </size>
       </property>
       <property name="movement" >
        <enum>QListView::Static</enum>
       </property>
       <property name="flow" >
        <enum>QListView::TopToBottom</enum>
       </property>
       <property name="isWrapping" stdset="0" >
        <bool>false</bool>
       </property>
       <property name="resizeMode" >
        <enum>QListView::Adjust</enum>
       </property>
       <property name="viewMode" >
        <enum>QListView::ListMode</enum>
       </property>
       <property name="uniformItemSizes" >
        <bool>false</bool>
       </property>
       <property name="batchSize" >
        <number>100</number>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
       <property name="currentRow" >
        <number>0</number>
       </property>
       <item>
        <property name="text" >
         <string>Appearance</string>
        </property>
        <property name="icon" >
         <iconset resource="images/images.qrc" >:/interface.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Playlist</string>
        </property>
        <property name="icon" >
         <iconset resource="images/images.qrc" >:/playlist.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Plugins</string>
        </property>
        <property name="icon" >
         <iconset resource="images/images.qrc" >:/plugins.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Advanced</string>
        </property>
        <property name="icon" >
         <iconset resource="images/images.qrc" >:/advanced.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="widget1" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <property name="bottomMargin" >
          <number>9</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox" >
           <property name="title" >
            <string>Skins</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>9</number>
            </property>
            <property name="topMargin" >
             <number>9</number>
            </property>
            <property name="rightMargin" >
             <number>9</number>
            </property>
            <property name="bottomMargin" >
             <number>9</number>
            </property>
            <item>
             <widget class="QListWidget" name="listWidget" >
              <property name="movement" >
               <enum>QListView::Static</enum>
              </property>
              <property name="flow" >
               <enum>QListView::TopToBottom</enum>
              </property>
              <property name="viewMode" >
               <enum>QListView::ListMode</enum>
              </property>
              <property name="modelColumn" >
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="plGroupBox" >
           <property name="title" >
            <string>Fonts</string>
           </property>
           <layout class="QGridLayout" >
            <property name="leftMargin" >
             <number>9</number>
            </property>
            <property name="topMargin" >
             <number>9</number>
            </property>
            <property name="rightMargin" >
             <number>9</number>
            </property>
            <property name="bottomMargin" >
             <number>9</number>
            </property>
            <property name="horizontalSpacing" >
             <number>6</number>
            </property>
            <property name="verticalSpacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="maximumSize" >
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Player:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="maximumSize" >
               <size>
                <width>80</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text" >
               <string>Playlist:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QLabel" name="plFontLabel" >
              <property name="frameShape" >
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text" >
               <string>???</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" >
             <widget class="QToolButton" name="plFontButton" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLabel" name="mainFontLabel" >
              <property name="frameShape" >
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow" >
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text" >
               <string>???</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <widget class="QToolButton" name="mainFontButton" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page" >
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <property name="bottomMargin" >
          <number>9</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QGroupBox" name="groupBox_3" >
           <property name="title" >
            <string>Metadata</string>
           </property>
           <layout class="QGridLayout" >
            <property name="leftMargin" >
             <number>9</number>
            </property>
            <property name="topMargin" >
             <number>9</number>
            </property>
            <property name="rightMargin" >
             <number>9</number>
            </property>
            <property name="bottomMargin" >
             <number>9</number>
            </property>
            <property name="horizontalSpacing" >
             <number>6</number>
            </property>
            <property name="verticalSpacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QCheckBox" name="metadataCheckBox" >
              <property name="text" >
               <string>Load metadata from files</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QGroupBox" name="groupBox_2" >
           <property name="title" >
            <string>Song Display</string>
           </property>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>9</number>
            </property>
            <property name="topMargin" >
             <number>9</number>
            </property>
            <property name="rightMargin" >
             <number>9</number>
            </property>
            <property name="bottomMargin" >
             <number>9</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Title format:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="formatLineEdit" />
            </item>
            <item>
             <widget class="QToolButton" name="titleButton" >
              <property name="text" >
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>121</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget" >
        <layout class="QGridLayout" >
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <property name="bottomMargin" >
          <number>9</number>
         </property>
         <property name="horizontalSpacing" >
          <number>6</number>
         </property>
         <property name="verticalSpacing" >
          <number>6</number>
         </property>
         <item row="1" column="0" >
          <widget class="QPushButton" name="preferencesButton" >
           <property name="text" >
            <string>Preferences</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QPushButton" name="informationButton" >
           <property name="text" >
            <string>Information</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>101</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="3" >
          <widget class="QTabWidget" name="pluginsTab" >
           <property name="tabPosition" >
            <enum>QTabWidget::North</enum>
           </property>
           <property name="tabShape" >
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex" >
            <number>0</number>
           </property>
           <widget class="QWidget" name="Input" >
            <attribute name="title" >
             <string>Input</string>
            </attribute>
            <layout class="QHBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="leftMargin" >
              <number>9</number>
             </property>
             <property name="topMargin" >
              <number>9</number>
             </property>
             <property name="rightMargin" >
              <number>9</number>
             </property>
             <property name="bottomMargin" >
              <number>9</number>
             </property>
             <item>
              <widget class="QTableWidget" name="inputPluginTable" >
               <property name="selectionMode" >
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior" >
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="rowCount" >
                <number>0</number>
               </property>
               <property name="columnCount" >
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="Output" >
            <attribute name="title" >
             <string>Output</string>
            </attribute>
            <layout class="QHBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="leftMargin" >
              <number>9</number>
             </property>
             <property name="topMargin" >
              <number>9</number>
             </property>
             <property name="rightMargin" >
              <number>9</number>
             </property>
             <property name="bottomMargin" >
              <number>9</number>
             </property>
             <item>
              <widget class="QTableWidget" name="outputPluginTable" >
               <property name="selectionMode" >
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior" >
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="fileDialogTab" >
            <attribute name="title" >
             <string>File Dialog</string>
            </attribute>
            <layout class="QVBoxLayout" >
             <property name="spacing" >
              <number>6</number>
             </property>
             <property name="leftMargin" >
              <number>9</number>
             </property>
             <property name="topMargin" >
              <number>9</number>
             </property>
             <property name="rightMargin" >
              <number>9</number>
             </property>
             <property name="bottomMargin" >
              <number>9</number>
             </property>
             <item>
              <layout class="QHBoxLayout" >
               <property name="spacing" >
                <number>6</number>
               </property>
               <property name="leftMargin" >
                <number>0</number>
               </property>
               <property name="topMargin" >
                <number>0</number>
               </property>
               <property name="rightMargin" >
                <number>0</number>
               </property>
               <property name="bottomMargin" >
                <number>0</number>
               </property>
               <item>
                <widget class="QComboBox" name="fileDialogComboBox" />
               </item>
               <item>
                <spacer>
                 <property name="orientation" >
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" >
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <property name="bottomMargin" >
          <number>9</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_4" >
           <property name="title" >
            <string>Tray Icon</string>
           </property>
           <layout class="QGridLayout" >
            <property name="leftMargin" >
             <number>9</number>
            </property>
            <property name="topMargin" >
             <number>9</number>
            </property>
            <property name="rightMargin" >
             <number>9</number>
            </property>
            <property name="bottomMargin" >
             <number>9</number>
            </property>
            <property name="horizontalSpacing" >
             <number>6</number>
            </property>
            <property name="verticalSpacing" >
             <number>6</number>
            </property>
            <item row="2" column="0" >
             <widget class="QCheckBox" name="toolTipCheckBox" >
              <property name="text" >
               <string>Show tooltip</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="messageCheckBox" >
              <property name="text" >
               <string>Show message</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="QSpinBox" name="messageDelaySpinBox" >
              <property name="minimum" >
               <number>100</number>
              </property>
              <property name="maximum" >
               <number>10000</number>
              </property>
              <property name="singleStep" >
               <number>100</number>
              </property>
              <property name="value" >
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0" >
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Message delay, ms:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2" >
             <widget class="QCheckBox" name="trayCheckBox" >
              <property name="text" >
               <string>Show tray icon</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="closeGroupBox" >
           <property name="title" >
            <string>Action On Close</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="leftMargin" >
             <number>9</number>
            </property>
            <property name="topMargin" >
             <number>9</number>
            </property>
            <property name="rightMargin" >
             <number>9</number>
            </property>
            <property name="bottomMargin" >
             <number>9</number>
            </property>
            <item>
             <widget class="QRadioButton" name="hideToTrayRadioButton" >
              <property name="text" >
               <string>Hide to tray</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="quitRadioButton" >
              <property name="text" >
               <string>Quit</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>341</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>510</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel" >
     <x>316</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolTipCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>344</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel" >
     <x>177</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>messageDelaySpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>208</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel" >
     <x>469</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trayCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>messageCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel" >
     <x>258</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>messageCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>messageDelaySpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>307</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel" >
     <x>469</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trayCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>closeGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>376</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel" >
     <x>436</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
